Tests/JavaKitMacroTests/JavaClassMacroTests.swift

```diff
@@ -78,6 +78,11 @@ class JavaKitMacroTests: XCTestCase {
             public init(javaHolder: JavaObjectHolder) {
                 self.javaHolder = javaHolder
             }
+
+            /// Casting to <JavaObject> will never be nil because <HelloWorld> extends it.
```

Comment:
```suggestion
            /// Casting to ``JavaObject`` will never be nil because ``HelloWorld`` extends it.
```

will actually result in proper type links, see here: https://www.swift.org/documentation/docc/linking-to-symbols-and-other-content